### ISSUE TITLE
[whishlist] provide xgettext to extract `` strings

### ISSUE THREAD

**Issue by mvo5** [gosexy/gettext#3]
Hi,

I use gosexy in one of my projects and I'm quite happy. One issue I noticed is that xgettext does not extract `` style strings. I created a custom go xgettext for this that uses the golang/ast module and that seems to make the extraction very simple. If thats something you are interessted in I can create a branch and send you a pull request, my current proof-of-concept is at http://bazaar.launchpad.net/~mvo/snappy/snapy-out-own-xgettext/view/head:/i18n/xgettext-go/main.go (but the real version would look a lot nicer :)

Cheers,
 Michael

**Comment by mvo5** [comment]
I updated the xgettext-go code since originally adding this issue and I think its in good shape now. Please let me know if it would be interessting to go-sexy and I can prepare a git branch for it.

**Comment by mvo5** [comment]
Any thoughts on this? If there is no interest in merging my xgettext-go here I will just split it out into a separate package.

**Comment by xiam** [comment]
Hello @mvo5,

I am tremendously sorry for this delay, at some point last year I turned off Github e-mail notifications and missed a lot of activity in this an other repos. I turned them on again late on the year but it seems like there are still many messages I didn't see on time.

Your proposal sounds awesome, I am not sure if you're still interested on creating this pull request or if you created a separate package from which I can pull some of the good stuff.

Thanks for your understanding,

José
